modules/scheduler/pre-existing-gke-cluster/main.tf

```diff
@@ -20,26 +20,51 @@ data "google_container_cluster" "existing_gke_cluster" {
   location = var.region
 }
 
-module "kubectl_apply" {
-  source = "../../management/kubectl-apply" # can point to github
-
-  cluster_id = data.google_container_cluster.existing_gke_cluster.id
-  project_id = var.project_id
+locals {
+  rdma_networks     = [for network_info in var.additional_networks : network_info if strcontains(upper(network_info.nic_type), "RDMA")]
+  non_rdma_networks = [for network_info in var.additional_networks : network_info if !strcontains(upper(network_info.nic_type), "RDMA")]
```

Comment:
Duplicate-diff pre-commit check fails. So, we need this code change here as well.